src/uct/ib/rc/base/rc_iface.c

```diff
@@ -695,10 +694,11 @@ ucs_status_t uct_rc_iface_qp_create(uct_rc_iface_t *iface, struct ibv_qp **qp_p,
     qp_init_attr.max_inl_recv        = iface->config.rx_inline;
 
     status = iface->super.ops->create_qp(&iface->super, &qp_init_attr, qp_p);
-    if (status == UCS_OK) {
-        *cap = qp_init_attr.cap;
+    if (status != UCS_OK) {
+        return status;
     }
 
+    *cap = qp_init_attr.cap;
```

Comment:
just noticed. why the status variable is static??
also, just 'return UCS_OK' below